### PR TITLE
Fixes #1267

### LINKED ISSUE #1267: Clean up TimestampsTrait
1.  scheduled_at is reduplicated. It is defined in both of TimestampsTrait and IncidentPresenter. Remove it from TimestampsTrait.
2.  move verified_at from  TimestampsTrait to it's own Presenter (SubscriberPresenter)

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

**Comment by phecho** [comment]
@GrahamCampbell @jbrooksuk 
How about these changes?
BTW, Travis is so fast now.

File: app/Presenters/SubscriberPresenter.php [hunk 1]
```diff
@@ -12,12 +12,24 @@
...
 
     /**
+     * Present formatted date time.
+     *
+     * @return string
+     */
+    public function verified_at()
+    {
+        return (new Date($this->wrappedObject->verified_at))
+            ->setTimezone($this->setting->get('app_timezone'))->toDateTimeString();
```
Comment (GrahamCampbell) [inline app/Presenters/SubscriberPresenter.php]: Please don't do this manual code.

Comment (GrahamCampbell) [inline app/Presenters/SubscriberPresenter.php]: Please use https://github.com/cachethq/Cachet/blob/master/app/Dates/DateFactory.php.

Comment (GrahamCampbell) [inline app/Presenters/SubscriberPresenter.php]: Oh, I see it was already like that. Hmm. I'll change this in another PR then.

**Pull request merged**